code/__HELPERS/trait_helpers.dm

```diff
@@ -244,6 +244,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NEVER_MISSES_DISPOSALS "trait_never_misses_disposals" // For janitors landing disposal throws
 #define TRAIT_SLEIGHT_OF_HAND "sleight_of_hand"
 #define TRAIT_KNOWS_COOKING_RECIPES "knows_cooking_recipes"
+#define TRAIT_ENLIGHTENED_HUMAN "is_enlightened_human"
```

Comment:
from first glance, this trait is really confusingly named. Please auto-document it :)

Reply:
I agree, this should be named something more like `TRAIT_XENOBIO_SPAWNED_HUMAN`. No one in game will ever see the name of this trait, so it should be named with an according name

Reply:
Resolving this, as the requested change has been made.